pom.xml

```diff
@@ -120,7 +120,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.7.5</version>
+        <version>3.1.1</version>
```

Comment:
Postgres uses 3.1.4 - should be identical?

Reply:
Yup